packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -133,6 +133,16 @@ export const getSingleOtherChatUser = (state: CommonState, chatId?: string) => {
   return getOtherChatUsers(state, chatId)[0]
 }
 
+export const getAllOtherChatUsers = createSelector(
+  [(state) => state, getChats],
```

Comment:
I could be wrong, but I think `state=>state` will negate the benefit of using `createSelector`, since the inputs to your compute function would change every time anything in the entire state changes?
I would say either use a proxy selector, or update `getOtherUsersFromChat` to take only the subset of the state it needs.

Reply:
Decided to just remove this altogether and live with a _slightly_ less polished loading UI.